composer.json

```diff
@@ -11,9 +11,15 @@
     ],
     "require": {
         "php": "^8.1.0",
-        "guzzlehttp/guzzle": "^7.5.0"
+        "php-http/discovery": "^1.15",
+        "php-http/multipart-stream-builder": "^1.2",
+        "psr/http-client-implementation": "^1.0.1",
+        "psr/http-client": "^1.0.1",
+        "psr/http-message": "^1.0.1"
     },
     "require-dev": {
+        "guzzlehttp/psr7": "^2.4.4",
```

Comment:
not needed since there is guzzle already

Reply:
Note that I'm not sure it's worth replacing this by php-http/multipart-stream-builder

Reply:
It's not possible to remove it because we require here newer version.